habitat_baselines/rl/ppo/ppo.py

```diff
@@ -146,9 +146,9 @@ def update(self, rollouts: RolloutStorage) -> Tuple[float, float, float]:
                     # Mask the loss. The mask corresponds to the transition right
                     # after a reset. This first observation needs to be ignored
                     # because its observation is actually part of the previous episode
-                    action_loss = action_loss * batch["masks"]
-                    value_loss = value_loss * batch["masks"]
-                    dist_entropy = dist_entropy * batch["masks"]
+                    action_loss = action_loss * batch["not_done_mask_0"]
```

Comment:
Only mask if the first mask is false as per https://github.com/facebookresearch/habitat-lab/pull/839/files#r840202226